.github/workflows/scheduled.yml

```diff
@@ -0,0 +1,29 @@
+name: scheduled
+on:
+  schedule:
+    # Runs at 00:00 UTC every Sunday, Tuesday, Thursday.
+    - cron: '0 0 * * 0,2,4'
+  workflow_dispatch:
+permissions:
+  contents: read
+  actions: write
+
+jobs:
+  trigger-workflow:
+    strategy:
+      matrix:
+        branch: ["main", "release-1.3"]
```

Comment:
Did you want to include 1.2.z as well, since it's still supported?

Reply:
I do, but I'd like to merge this first to see if it works and iron out any issues (it works on my fork but I'm not 100% sure it will work here).

Once 1.3 is working fine, I'll add 1.2 (which will need something similar to #4763 first, too).